configure

```diff
@@ -298,7 +318,7 @@ if [ "${ARCH}" = "aarch64" ] && [ "${SM_VSN}" = "60" ]; then
   exit 1
 fi
 
-if [ "${ERLANG_OS}" = "unix" ]; then
+if [ "${JS_ENGINE}" = "spidermonkey" ] && [ "${ERLANG_OS}" = "unix" ]; then
```

Comment:
This check should not be skipped unless support for SpiderMonkey is completely disabled.  That is, this part has to be modified in lockstep with the part that is responsible for building SM.  If that is not deactivated, this should not be either.

Reply:
You're right. Good find! I'll fix it

Reply:
This should be fixed in the last update. If spidermonkey is enabled, we should run the library/header checks.